pkg/interactive-installer/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+
+ENV BUILD_PKGS curl build-base git ncurses-dev musl-dev python3 bash
+ENV PKGS ncurses-dev smartmontools mtools strace util-linux e2fsprogs e2fsprogs-extra jq
+
+RUN eve-alpine-deploy.sh
+ENV RUSTUP_HOME=/usr/local
+ENV CARGO_HOME=/usr/local/cargo
+ENV CARGO_TARGET_DIR=/usr/local/my-installer/target
+ENV RUST_VERSION=1.79.0
+ADD https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-musl.tar.gz rust-${RUST_VERSION}-x86_64-unknown-linux-musl.tar.gz
+ADD https://static.rust-lang.org/dist/rust-${RUST_VERSION}-aarch64-unknown-linux-musl.tar.gz rust-${RUST_VERSION}-aarch64-unknown-linux-musl.tar.gz
+
+# Install Rust using the standalone installer
+RUN ARCH="aarch64" && \
+    if [ "$(uname -m)" == "x86_64" ]; then \
+        ARCH="x86_64"; \
+    fi && \
+    tar -xzf rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl.tar.gz && \
+    cd rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl && \
+    ./install.sh && \
+    cd .. && \
+    rm -rf rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl.tar.gz
+
+ENV PATH "$PATH:/usr/local/cargo/bin"
+
+WORKDIR /usr/local/my-installer
+COPY . .
```

Comment:
Is this a left-over?

Reply:
If you are referring to "COPY . ." it is not a leftover. It copies all local files to the container.

Reply:
But it even copies the `Dockerfile` into the container, doesn't it?
And the implication of that is, that if you add something at the end of the Dockerfile, it will on the next build of the container start from here again.

Also you copy the vendor directory again at https://github.com/lf-edge/eve/pull/4055/files#diff-bfb0a10960fc27681a077dedc90580981dd639615e9e16de2b6917e942a951b1R33 .

Reply:
This is an intermediate stage; hence, all files will be discarded. However, I changed it to copy only the required files.